Server/src/models/AdminManagementModel.ts

```diff
@@ -0,0 +1,76 @@
+import { NotFound } from "@tsed/exceptions";
+import { Connection, getConnection } from "typeorm";
+import { Accounts, selectAccountIdFromEmailQuery } from "./entities/Accounts";
+
+
+
+export class AdminManagementModel {
+    private connection: Connection;
+    constructor() {
+        this.connection = getConnection();
+    }
+
+    async fetchAccountIdFromEmail(userEmail: string) {
+        let userRawData = await selectAccountIdFromEmailQuery(this.connection, userEmail)
+        if (userRawData == undefined)

```

Comment:
could be if(userRawData) {
return userRawData.id
} else {
throw 404
}

Reply:
https://stackoverflow.com/questions/19839952/all-falsey-values-in-javascript

Reply:
This is an extremely good catch; I was not aware of the concept of falsy. I'll implement it here and it may be wise to create an enhancement later to change all similar instances of this in the backend to follow the format here.